src/pages/proofRequest/index.tsx

```diff
@@ -17,8 +17,7 @@ export const ProofRequest = () => {
         }}
         title={verification?.name || ''}
         subtitle={verification?.description || ''}
-        shortLink={connectUrl}
-        loading={loading}
+        shortLink={verification?.connection_url}
```

Comment:
I think it's better to make a condition for showing this modal if it has short link
if we don't have a connect URL what is the purpose of showing this modal?

Reply:
It is used for preview, the short link is not created yet, and we just use a fade image instead of QR code